web-components/src/types/news-feed.ts

```diff
@@ -0,0 +1,48 @@
+export interface NewsData {
```

Comment:
@teolemon did you check the format is the correct one ?

```suggestion
/**
 * This is the format of the JSON file where data is stored
 */
export interface NewsData {
```